examples/async/fastapi_example.py

```diff
@@ -1,13 +1,42 @@
+from contextlib import asynccontextmanager
+
 import async_module
 import fastapi
+import hamilton_sdk.adapters
```

Comment:
We try to do module imports...
```suggestion
from hamilton_sdk import adapters 
```

Reply:
This is throwaway code